### ISSUE TITLE
Abstract/Support the abstraction of schema/data migrations.

### ISSUE THREAD

**Issue by ktstowell** [liveaxle/node-pg-migrate#5]
I think what I'm going to work on next is that when you create a migration, two files are created:

```
/migrations
-/schema
--/01.foo.migration.js
-/data
--/01.foo.migration.js
```

And then change the tasks to be up:schema/up:data, etc.

To couple them is to be very assuming of the migrations that need to be done and often, the act of a data migration happens after you've made schema changes, typically in the form of exporting data out of the old db/table and putting it into a new one.

**Comment by ktstowell** [comment]
@adamison @jcroft-pdx what are your thoughts?

**Comment by ktstowell** [comment]
Solved in: #10
